### ISSUE TITLE
Transparent card background

### ISSUE THREAD

**Issue by mierz00** [LeoLeBras/react-router-navigation#64]
Thanks for the great package! 

I have a login screen with an image as a background which I would like to render the different inputs over, like this.
``` 
      <Wallpaper>
        <Logo />
        <Navigation
          cardStyle={{ backgroundColor: 'transparent', shadowOpacity: 0 }}
          hideNavBar
        >
          <Card cardStyle={{ backgroundColor: 'transparent', shadowOpacity: 0 }} exact path="/login" render={_UserSection} />
          <Card cardStyle={{ backgroundColor: 'transparent', shadowOpacity: 0 }} path="/accounts" render={() => <AccountSelect />} />
          <Card cardStyle={{ backgroundColor: 'transparent', shadowOpacity: 0 }}  path="/password" render={() => <PasswordSection />} />
        </Navigation>
      </Wallpaper>
```
However, I am not able to get rid of a light grey background which seems to have appeared after using `react-router-navigation`. So, in the first case, Is it possible to have transparent backgrounds on the cards? If so, does anyone know of any way of removing this grey background from the app. 

**What is the current behavior?**
![example](https://image.ibb.co/kTicAx/Capture_d_e_cran_2018_02_23_a_9_12_32_PM.png)
Not able to remove grey background colour on cards. 

1. Added ` cardStyle={{ backgroundColor: 'transparent' }} ` on every `<Navigation />` 
2. Checked all the styles in my app. Yet, I haven't found where this background grey is coming from.
I also have the same problem in this gif. Which was in a previous issue. 
<img src="https://cloud.githubusercontent.com/assets/8975443/24124196/d0583c28-0dc2-11e7-8945-2b1922df848c.gif" width="300">

Although I am not able to change cardStyle properties.

I am trying to achieve something like @CharlesMangwa login screen,
<img src="https://cdn-images-1.medium.com/max/800/1*syDzYEx0rc7WcEoyaUBd4Q.gif" width="300">

**Environment (include versions)** `Vscode`
* Device: `iPhone 6 (Sim)`
* OS: `IOS`
* React-Native (version): `0.53.3`
* React-Router (version): `4.2.0`
* React-Router-Navigation (version): `1.0.0-rc.4`

**Comment by skyskysky11** [comment]
Same here. Any solution?

**Comment by mierz00** [comment]
@skyskysky11 
No sorry, I ended up creating a custom animation wrapper using LayoutAnimation which I wrapped around each route in a switch.

**Comment by LeoLeBras** [comment]
Should work with https://github.com/LeoLeBras/react-router-navigation/pull/68 😎

**Comment by LeoLeBras** [comment]
This should be fixed in `1.0.0-rc.5`
